.github/workflows/scorecards.yml

```diff
@@ -0,0 +1,69 @@
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: '45 9 * * 3'
```

Comment:
Any particular reason for this value? It would be nice to add a comment ~ `# Every Wednesday at 9:45AM`

Reply:
I'd very much recommend we don't pick a time that's a specific human nice round number thing like "45" minutes. It's when there's many other crons too. Better to pick something actually random instead :smile:. 

But I also agree with @mattlord's later comment, do we need a cron at all? Is it too expensive to run on each push to main?

Reply:
@mattlord  @dbussink  The time is generated by the scorecard github action. 
The action itself is not expensive, but running this action on every push to the main may make it expensive. 